apps/web/src/blocks/Layout/index.tsx

```diff
@@ -0,0 +1,92 @@
+'use client';
+
+import React, { PropsWithChildren } from 'react';
+import Image from 'next/image';
+import Link from 'next/link';
+import { css, styled } from 'styled-components';
+
+import Button from 'ui/components/Button';
+
+import { Layout as LayoutType, Media } from '@web/payload/payload-types';
+import isExpandedDoc from '../../lib/isExpandedDoc';
+
+const Wrapper = styled.div`
+  display: flex;
+  flex-direction: column;
+  gap: 20px;
+  justify-content: space-between;
+  align-items: center;
+  padding: 30px;
+  ${({
+    theme: {
+      mq: { lg }
+    }
+  }) => lg`
+    flex-direction: row;
+    gap: unset;
+  `}
+`;
+
+const MenuLinksContainer = styled.div`
+  display: flex;
+  align-items: center;
+  gap: 28px;
+`;
+
+export const MenuLink = styled.p`
+  ${({ theme: { colors } }) => css`
+    color: ${colors.white};
+  `}
+  && {
+    font-size: 17px;
+    font-style: normal;
+    text-decoration: none;
+    font-weight: 400;
+    line-height: normal;
+    letter-spacing: -0.17px;
+    margin: 0;
+    padding: 0;
+  }
+`;
+
+function MenuLinks({ links }: { links: LayoutType['headerItems'] }) {
+  return (
+    <MenuLinksContainer>
+      {links?.map((link) =>
+        link?.type === 'button' ? (
+          <Button key={`FooterLink-${link?.title}`}>{link?.title}</Button>
+        ) : (
+          <Link key="MenuLink" href={link?.url}>
+            <MenuLink>{link?.title}</MenuLink>
+          </Link>
+        )
+      )}
+    </MenuLinksContainer>
+  );
+}
+
+function Layout({
+  logo,
+  footerItems,
+  headerItems,
+  children
+}: PropsWithChildren<LayoutType>) {
+  return (
+    <>
+      <Wrapper>
+        {isExpandedDoc<Media>(logo) && (
```

Comment:
what happens if this is false?

Reply:
Doesn't render the logo and creates an odd UI state where the nav links are pushed to the left. Fixed by adding an empty self closing div to fulfill the requirements of the flexbox.